CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Changelog
 
+## streamlink 1.3.0 (2019-11-21)
+
+A new release with plugin updates and fixes, including Twitch.tv (see #2680), which had to be delayed due to back and forth API changes.
```

Comment:
I don't think the changelog links directly to issues right? So it needs to be `(see [#2680](https://github.com/streamlink/streamlink/issues/2680))` for any references to the issue.

Reply:
The github release pages use the same github flavored markdown as comments on issue trackers and issue- or commit-ids will get turned into links automatically. The rendered markdown on the git tree however does not.

Let me change this real quick...